server/db/migrations/20_create_grid_service_instance.rb

```diff
@@ -1,7 +1,12 @@
 class CreateGridServiceInstance < Mongodb::Migration
   def self.up
     GridServiceInstance.create_indexes
-    Container.unscoped.where(container_type: 'container').includes(:grid_service).each do |c|
+    migrate_containers('volume')
+    migrate_containers('container')
```

Comment:
Thinking about ordering... can there be multiple volume/containers for a given service instance across different host nodes, and the `GridServiceInstance` might get be created with a different host node depending on the ordering of the containers here?

With `volume` containers first... could there be old volume containers left over from stateful services that got rescheduled away from an offline node, and this migration might revert the container from where it is currently running, back to some old node where it used to be running?

Should we migrate the `container` or `volume` types first, or sort by something like the `Container.updated_at`?